web/package.json

```diff
@@ -81,8 +81,6 @@
     "cy:run": "cypress run",
     "cy:ci": "cypress run --parallel --record --key $CYPRESS_RECORD_KEY",
     "prettier": "prettier --write ./src",
-    "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public",
```

Comment:
Sad to see the storybook getting rid of the project (I used it a lot in other projects before to test component behaviors), but right now, that makes sense.
Today it is easier to build a component and test it with our UI pointing to an example API than creating a story and mocking many things.

Reply:
Exactly, storybook is great but is not useful at this point, in the future if makes sense we can add it back again